.github/workflows/publish_to_dockerhub.yml

```diff
@@ -12,7 +12,7 @@ env:
   DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
 
 jobs:
-  push-image:
+  push-fidesops-image:
```

Comment:
As discussed — would be good to move this into a separate workflow so we don't publish the privacy centre on every release if there are no changes.